tox.ini

```diff
@@ -5,15 +5,16 @@ small_test_ee = quay.io/ansible/python-base
 mypy_tmp_dir = mypy_tmp_dir
 
 [tox]
-envlist = linters, type, py38, report, clean
-minversion = 3.16.1
+envlist = linters, type, py{36,37,38,39,310}, report, clean
+# we do not want debug old bugs
+minversion = 3.24.3
```

Comment:
I feel like this is a separate thing to change. I don't see why this arbitrary version is chosen but it's not a "fix" as the PR title suggests. If you want to bump it, do so in a PR that separately demonstrates that there are no problems with this bump. OTOH, I would use the first version that drops Python 3.5 support unless there are legitimate tox features (like new substitutions) that are really used in the config. If you want to add those new features, it would be expected that the min version would be set in a respective PR.